src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,31 @@
+// prop-types
+import Proptypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+// css
+import "./button.css";
+
+const Button = (props) => {
+  const { btnText, handleClick, className } = props;
```

Comment:
Add button type too, because if you want to add custom styles to only some button then those can be send from parent using className props